components/dashboard/src/data/organizations/org-settings-query.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { TeamSettings } from "@gitpod/gitpod-protocol";
+import { useQuery } from "@tanstack/react-query";
+import { getGitpodService } from "../../service/service";
+
+export type TeamSettingsResult = TeamSettings;
+
+type Args = {
+    teamId: string;
+};
+
+export const useOrgSettingsQuery = ({ teamId }: Args) => {
+    return useQuery<TeamSettingsResult>({
+        queryKey: getOrgSettingsQuery(teamId),
+        staleTime: 1000 * 60 * 1, // 1 minute
```

Comment:
Unless there's a specific need for this to have a custom staleTime of 1 minute, I'd suggest we leave it off and use the default (stale immediately, will revalidate next time it's shown).

Reply:
I leave it to @mustard-mh since I don't know a context